### PR TITLE
Get unit data by searching directory

### PR DESCRIPTION
**Pull request by yeahross0**

### CODE REVIEW

**Comment by RM35** [comment]
I just added this

File: world/world.gd [hunk 1]
```diff
@@ -47,6 +43,28 @@ func _process(delta):
...
 	$Debug/Panel/MarginContainer/VB/State.text = "STATE: " + str(game_state)
 
+# Non recursively searches the directory
+func files_with_extension(directory_path: String, extension: String) -> Array:
+	var card_types := []
+	var card_directory := Directory.new()
+	
+	extension = extension.trim_prefix(".")
+
+	var result := card_directory.open(directory_path)
```
Comment (willpxxr) [inline world/world.gd:53]: nice leak

**Comment by RM35** [comment]
Already added in #38

**Pull request closed without merging**